charts/aad-pod-identity/templates/mic-deployment.yaml

```diff
@@ -3,10 +3,7 @@ kind: Deployment
 metadata:
   name: {{ template "aad-pod-identity.mic.fullname" . }}
   labels:
-    app.kubernetes.io/name: {{ template "aad-pod-identity.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    helm.sh/chart: {{ template "aad-pod-identity.chart" . }}
+    {{- include "aad-pod-identity.labels" . | nindent 4 }}
```

Comment:
These same labels are used in the [selector.matchLabels](https://github.com/Azure/aad-pod-identity/blob/master/charts/aad-pod-identity/templates/mic-deployment.yaml#L15-L19). Can you please update that too in both the mic-deployment and nmi-daemonset?

Reply:
Done